src/code/z_player_lib.c

```diff
@@ -1076,132 +1148,93 @@ void Player_DrawGetItem(GlobalContext* globalCtx, Player* this) {
+    posOffset.z = 0.0f;
+    Matrix_MultVec3f(&posOffset, &posA);
+    posOffset.z = arg2;
+    Matrix_MultVec3f(&posOffset, &posB);
 
     if (1) {}
 
-    if (BgCheck_AnyLineTest3(&globalCtx->colCtx, &sp8C, &sp80, &sp74, &sp9C, 1, 1, 1, 1, &bgId)) {
+    if (BgCheck_AnyLineTest3(&globalCtx->colCtx, &posA, &posB, &aimProjectedPos, &polygon, 1, 1, 1, 1, &bgId)) {
+        Vec3f screenPos;
+        f32 w;
+        f32 scale;
+
         OPEN_DISPS(globalCtx->state.gfxCtx, "../z_player_lib.c", 2572);
 
         OVERLAY_DISP = Gfx_CallSetupDL(OVERLAY_DISP, 0x07);
 
-        SkinMatrix_Vec3fMtxFMultXYZW(&globalCtx->mf_11D60, &sp74, &sp68, &sp64);
+        SkinMatrix_Vec3fMtxFMultXYZW(&globalCtx->mf_11D60, &aimProjectedPos, &screenPos, &w);
 
-        sp60 = (sp64 < 200.0f) ? 0.08f : (sp64 / 200.0f) * 0.08f;
+        scale = (w < 200.0f) ? 0.08f : (w / 200.0f) * 0.08f;
 
-        Matrix_Translate(sp74.x, sp74.y, sp74.z, MTXMODE_NEW);
-        Matrix_Scale(sp60, sp60, sp60, MTXMODE_APPLY);
+        Matrix_Translate(aimProjectedPos.x, aimProjectedPos.y, aimProjectedPos.z, MTXMODE_NEW);
+        Matrix_Scale(scale, scale, scale, MTXMODE_APPLY);
 
         gSPMatrix(OVERLAY_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_player_lib.c", 2587),
                   G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
         gSPSegment(OVERLAY_DISP++, 0x06, globalCtx->objectCtx.status[this->actor.objBankIndex].segment);
-        gSPDisplayList(OVERLAY_DISP++, D_0602CB48);
+        gSPDisplayList(OVERLAY_DISP++, gLinkHookshotPointerDL);
 
         CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_player_lib.c", 2592);
     }
 }
 
-Vec3f D_801260D4 = { 1100.0f, -700.0f, 0.0f };
-
-f32 sSwordLengths[] = {
-    0.0f, 4000.0f, 3000.0f, 5500.0f, 0.0f, 2500.0f,
-};
-
-Gfx* sBottleDLists[] = { 0x0602AD58, gLinkChildBottleDL };
-
-Color_RGB8 sBottleColors[] = {
-    { 255, 255, 255 }, { 80, 80, 255 },   { 255, 100, 255 }, { 0, 0, 255 }, { 255, 0, 255 },
-    { 255, 0, 255 },   { 200, 200, 100 }, { 255, 0, 0 },     { 0, 0, 255 }, { 0, 255, 0 },
-    { 255, 255, 255 }, { 255, 255, 255 }, { 80, 80, 255 },
-};
-
-Vec3f D_80126128 = { 398.0f, 1419.0f, 244.0f };
-
-BowStringData sBowStringData[] = {
-    { 0x0602B108, { 0.0f, -360.4f, 0.0f } },  // bow
-    { 0x060221A8, { 606.0f, 236.0f, 0.0f } }, // slingshot
-};
-
-Vec3f D_80126154[] = {
-    { -4500.0f, -3000.0f, -600.0f },
-    { 1500.0f, -3000.0f, -600.0f },
-    { -4500.0f, 3000.0f, -600.0f },
-    { 1500.0f, 3000.0f, -600.0f },
-};
-
-Vec3f D_80126184 = { 100.0f, 1500.0f, 0.0f };
-Vec3f D_80126190 = { 100.0f, 1640.0f, 0.0f };
-
-Vec3f D_8012619C[] = {
-    { -3000.0f, -3000.0f, -900.0f },
-    { 3000.0f, -3000.0f, -900.0f },
-    { -3000.0f, 3000.0f, -900.0f },
-    { 3000.0f, 3000.0f, -900.0f },
-};
-
-Vec3f D_801261CC = { 630.0f, 100.0f, -30.0f };
-Vec3s D_801261D8 = { 0, 0, 0x7FFF };
-
-Vec3f D_801261E0[] = {
-    { 200.0f, 300.0f, 0.0f },
-    { 200.0f, 200.0f, 0.0f },
-};
+typedef struct {
+    /* 0x00 */ void* dList;
```

Comment:
dlist can be typed as Gfx*. It was probably void before out of a precautionary measure, but we know for sure it cant be anything else

Reply:
(this is bow string info, i forgot to include that context in the comment)